docs/Reference.md

```diff
@@ -112,13 +112,14 @@ Clients may append more information and metadata to the _end_ of this string as
 
 The HTTP API implements a process for limiting and preventing excessive requests in accordance with [RFC 6585](https://tools.ietf.org/html/rfc6585#section-4). API users that regularly hit and ignore rate limits will have their API keys revoked, and be blocked from the platform. For more information on rate limiting of requests, please see the [Rate Limits](#DOCS_RATE_LIMITS/rate-limits) section.
 
->warn
->A bot account must connect and identify to a [Gateway](#DOCS_GATEWAY/connecting) at least once before being able to send messages.
-
 ## Gateway (WebSocket) API
 
 Discord's Gateway API is used for maintaining persistent, stateful websocket connections between your client and our servers. These connections are used for sending and receiving real-time events your client can use to track and update local state. The Gateway API uses secure websocket connections as specified in [RFC 6455](https://tools.ietf.org/html/rfc6455). For information on opening Gateway connections, please see the [Gateway API](#DOCS_GATEWAY/gateways) section.
 
+>warn
+>A bot must connect to and identify with a gateway at least once it can use the [Create Message](#DOCS_CHANNEL/create-message) endpoint.
```

Comment:
Nitpicking here, this part seems a engrish-y now. The original way (`A bot must connect and identify to a gateway at least once before being able to use the [Create Message](#DOCS_CHANNEL/create-message) endpoint.`) seemed just fine in my view.

Reply:
🙃 I seem to have misplaced a `before`.